R/quantity.R

```diff
@@ -88,6 +106,9 @@ Quantity <- R6::R6Class(
     },
     allUnits = function() {
       rClr::clrCallStatic(WITH_DIMENSION_EXTENSION, "AllUnitNames", self$ref)
+    },
+    reset = function() {
+      self$isFixedValue <- FALSE

```

Comment:
@msevestre Can you explain this? What exactly does it mean and when is it set to TRUE`?

Reply:
Setting it to FALSE means that we forget the value set by the user and we use the formula value instead.
Setting it to TRUE should not be done normally by the user. This flag should actually be a readonly flag in my opinion

That's why I have created a reset method instead to hide the implementation details

reset => isFixedValue = FALSE => we use the formula instead of the value that may have been set by the user 

Note. This flag has no meaning for a constant parameter